.openpublishing.redirection.json

```diff
@@ -2851,6 +2851,14 @@
             "source_path": "docs/desktop-wpf/xaml-services/xuid-directive.md",
             "redirect_url": "/dotnet/desktop/xaml-services/xuid-directive"
         },
+        {
+            "source_path": "docs/fundamentals/code-analysis/style-rules/ide0003.md",
+            "redirect_url": "/dotnet/fundamentals/code-analysis/style-rules/ide0003_ide0009.md"
```

Comment:
Redirects needed when multiple complimentary rule IDs are clubbed with same option/option group and hence need to be in the same doc page. These redirects will allow each analyzer to specify `docs/fundamentals/code-analysis/style-rules/<%RuleId%>.md` by directly mapping rule ID to URI.